docs/migrations.rst

```diff
@@ -86,4 +86,4 @@ to ensure we can rebuild the view without issue.
 Migration Patterns and Best Practices
 -------------------------------------
 
-- `Auto-Managed Migration Pattern <migration_command_pattern>`_.
+- :ref:`auto-managed-migration-pattern`
```

Comment:
@proteusvacuum i think we were trying to figure this out a couple week ago. fyi in case you're still curious

Reply:
I tried many variations of this but couldn't get it to work... happy someone figured out the right syntax.

Reply:
The [docs build failed](https://readthedocs.org/projects/commcare-hq/builds/8744531/), so I don't know if it works yet.

Reply:
You can test locally with:
```
$ pip install -r requirements/docs-requirements.txt
$ cd docs
$ make html
```

Reply:
build passed and it works